### DISCUSSION TITLE
Is it possible to send signed and encrypted emails? (S/MIME Certificates)

### DISCUSSION THREAD

**Discussion by TheMultii** [laravel/framework#40692] [Q&A] [answered]
I cannot find this in the laravel documentation. So I am wondering if this possibility exists. Many email clients support S/MIME. And I would like to have a "verified email address" badge in e.g. Gmail. 

https://en.wikipedia.org/wiki/S/MIME
https://www.actalis.com/s-mime-certificates.aspx

**Comment by josiahke** [comment, marked as answer]
see this 

https://medium.com/@linushstge/laravel-s-mime-mails-93da3e76fb93

**Reply by xiaohuilam** [reply]
It's a great solution, but only cannot use with telescope's mail logging. Or it throws 
```
Cannot traverse an already closed generator
```
